wallet-core/src/notes/pick.rs

```diff
@@ -13,7 +13,7 @@ use crate::notes::MAX_INPUT_NOTES;
 use execution_core::transfer::phoenix::{NoteLeaf, ViewKey as PhoenixViewKey};
 use execution_core::BlsScalar;
 
-/// Pick the notes to be used in a transaction from a vector of notes.
+/// Pick the notes to be used in a transaction from an owned notes list.
```

Comment:
Very picky nit and feel free to ignore, but I would say
```suggestion
/// Pick the notes to be used in a transaction from a list of owned notes.
```